src/ui.data-table/index.stories.js

```diff
@@ -195,32 +196,6 @@ const EmptyTemplate = (args) => ({
   `,
 });
 
-const SlotTemplate = (args) => ({
-  components: { UTable, UButton, ULink, UMoney, UBadge, URow },
-  setup() {
-    return { args };
-  },
-  template: `
-    <UTable
-      v-bind="args"
-      :rows="itemsData"
-    >
-      ${args.slotTemplate}
-    </UTable>
-  `,
-  computed: {
-    itemsData() {
```

Comment:
`itemsData()` should be present to manipulate rows amount

Reply:
`itemsData()` is present in DefaultTemplate (this is deleted SlotTemplate, as they were combined), manipulation of rows works fine, just as before